hexrdgui/resources/calibration/default_fiddle_config.yml

```diff
@@ -0,0 +1,15 @@
+beam:
+  energy: 10.25
+  vector: {azimuth: -92.99696498922033, polar_angle: 114.98012133882347}
+detectors:
+  default:
+    pixels:
+      columns: 512
+      rows: 1024
+      size: [0.025, 0.025]
+    transform:
+      tilt: [1.8496392739293273, 2.5380251768660202, -0.01561236535824728]
+      translation: [-46.44506775, -1.82675053, 56.74914575]
+oscillation_stage:
+  chi: 0.0
+  translation: [0.0, 0.0, 0.0]
```

Comment:
Is this file needed, considering we have one with all of the detectors in `hexrd`? If we still need this, it's fine. I'm just checking...

Reply:
This approach could certainly be re-thought a little and corrected to use the `hexrd` files, but it would require some changes to the logic. I think it requires enough changes that it shouldn't be done on this branch, but I'd be happy to open this as an issue and put up a new PR for it!